### PR TITLE
Performance meeting updates

### PR DESCRIPTION
**Pull request by psteckler**
Added assessments for each open example based on discussions with INRIA (Maxime, Matthieu, PMP). Also distinguished partially-fixed from fixed examples.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,48 +76,108 @@ Some examples of Coq being really slow:
...
   [`slow_lets.v`](./slow_lets.v) for an example of exponential
   behavior.
 
-## Already fixed or partially fixed issues:
+  **Assessment**: maybe the let-unfolding could be done lazily, or
+  maybe it's not really a bug; or maybe the fix is just to remove the
+  code
+
+  [Bug #4642](https://coq.inria.fr/bugs/show_bug.cgi?id=4642) - 
+  `cbv [some identifiers]` can run through 64 GB of RAM in 15 minutes
```
Comment (JasonGross) [inline README.md]: Please add a reference to `[`slow_fiat_examples/README.md`](./slow_fiat_examples/README.md) for
    more details and instructions on running.`, as was in the original mention

File: README.md [hunk 2]
```diff
@@ -56,48 +76,108 @@ Some examples of Coq being really slow:
...
   [`slow_failing_setoid_rewrite.v`](./slow_failing_setoid_rewrite.v)
 
+  **Assessment**: Ltac2 will allow such termination via a non-backtracking
+  exception mechanism, tentatively named "panic"
+
 - [Bug #4636](https://coq.inria.fr/bugs/show_bug.cgi?id=4636) - `set
   (x := y)` can be 100x slower than `pose y as x; change y with x` -
   see [`slow_set.v`](./slow_set.v).  (The reverse can also happen,
   where `change` is orders of magnitude slower than `set`.  See
   [bug #4779](https://coq.inria.fr/bugs/show_bug.cgi?id=4779) in the next
```
Comment (JasonGross) [inline README.md]: Please update this to mention that 4779 has been moved down

File: README.md [hunk 3]
```diff
@@ -56,48 +76,108 @@ Some examples of Coq being really slow:
...
+  maybe it's not really a bug; or maybe the fix is just to remove the
+  code
+
+  [Bug #4642](https://coq.inria.fr/bugs/show_bug.cgi?id=4642) - 
+  `cbv [some identifiers]` can run through 64 GB of RAM in 15 minutes
+
+  **Assessment**: NOT DONE
+
+  [Bug #4779](https://coq.inria.fr/bugs/show_bug.cgi?id=4779) -
+  `pose y as x; change y with x in H` can be 1300x slower than `set (x := y) in H`
```
Comment (JasonGross) [inline README.md]: Please add a reference to `[`slow_fiat_examples/README.md`](./slow_fiat_examples/README.md) for
    more details and instructions on running.`, as was in the original mention

**Review by JasonGross — CHANGES_REQUESTED** [review]
Thanks!  If you make the changes I requested, I'll merge.

File: README.md (hunk 3 above)
Comment (psteckler) [reply README.md]: The changes you suggested should be done, please check them.

**Review by JasonGross — APPROVED** [review]
Thanks!

**Pull request merged**